docs/search_relevance/query_design.md

```diff
@@ -78,6 +103,7 @@ Searching for a work based on local and external identifiers e.g. Catalogue API
 * Searching for a list of identifiers, I get all the results back
 * Searches should be case insensitive
 * If the search query contains and ID and other input, we should match
+
```

Comment:
?

Reply:
Not sure how this crept in... I assume it's gitbook autoformatting?